app/src/Components/Event/EventReturnedList.tsx

```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components/native'
 import { FlatList } from 'react-native'
 import EventSummary from './EventSummary'
 import { Events } from '@/Services/modules/events'
+import { navigate } from '@/Navigators/utils'
```

Comment:
Don't think you need this as it's not being used in this file?

Reply:
Thought I'd removed these, must have rolled back a commit at some point!

Reply:
😄  I think this could still be removed as isn't in use?